widget/src/pick_list.rs

```diff
@@ -637,20 +684,22 @@ pub fn draw<'a, T, Renderer>(
         renderer.fill_text(Text {
             content: label,
             size: text_size,
-            font: font.clone(),
+            line_height: text_line_height,
+            font,
             color: if is_selected {
                 style.text_color
             } else {
                 style.placeholder_color
             },
             bounds: Rectangle {
                 x: bounds.x + padding.left,
-                y: bounds.center_y() - text_size / 2.0,
+                y: bounds.center_y(),
                 width: bounds.width - padding.horizontal(),
-                height: text_size,
+                height: text_size * 1.2,
```

Comment:
Hard-coded `1.2` line height value is used here when we have `text_line_height` that should be used

Reply:
Fixed in f0c87375d53d620b5939d87e1bc54288f9c184ac.

Reply:
@hecrj Just a heads up, this only addressed the icon height, but not the height / bounds of the actual text (last fill_text of the draw helper)

Reply:
:facepalm: Fixed in 1400b5187d6fa30f5fa4f83684939f6dc87cb55f.